### ISSUE TITLE
nhost init --remote gives interface conversion panic error

### ISSUE THREAD

**Issue by phlogios** [nhost/cli#627]
I started a new app with nhost, and connected it to an existing github repo that didn't contain any nhost files.

I did: `nhost init --remote` in the root folder of the github repo, and selected the nhost app.

This error was displayed:
```
> Creating your app...panic: interface conversion: interface {} is nil, not []interface {}

goroutine 1 [running]:
github.com/nhost/cli/hasura.(*Client).GetSchemas(0x0?)
        /home/runner/work/cli/cli/hasura/hasura.go:210 +0x7fe
github.com/nhost/cli/cmd.pullMigration(0xc000c57c30, {0x12df7ac, 0x4})
        /home/runner/work/cli/cli/cmd/pull.go:161 +0x205
github.com/nhost/cli/cmd.glob..func13(0x1ebf320?, {0xc000793290?, 0x0?, 0x1?})
        /home/runner/work/cli/cli/cmd/init.go:253 +0x106e
github.com/spf13/cobra.(*Command).execute(0x1ebf320, {0xc000793270, 0x1, 0x1})
        /home/runner/work/cli/cli/vendor/github.com/spf13/cobra/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x1ebfea0)
        /home/runner/work/cli/cli/vendor/github.com/spf13/cobra/command.go:1044 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/work/cli/cli/vendor/github.com/spf13/cobra/command.go:968
github.com/nhost/cli/cmd.Execute()
        /home/runner/work/cli/cli/cmd/root.go:83 +0x25
main.main()
        /home/runner/work/cli/cli/main.go:31 +0x17
```

The folder was still created, but I can't run `nhost up` because it complains that the postgres container is unhealthy. I have no idea if the errors are related to each other. That specific error is already reported here: https://github.com/nhost/cli/issues/460

I'm running on WSL2, Ubuntu 22.04 (also tried Ubuntu 20.04 LTS under WSL2), with latest stable Docker for Windows with WSL2 mode enabled.

**Comment by vrenan** [comment]
I also got this error.

**Comment by elephant3** [comment]
Should be fixed in the latest v0.9.1, please upgrade
